### PR TITLE
fix: node success messages

### PR DESCRIPTION
**Pull request by Benehiko**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Review by aeneasr — APPROVED** [review]

**Comment by Benehiko** [comment]
Added storybook for verification success
image

File: src/react-components/ory/helpers/error-messages.tsx [hunk 1]
```diff
@@ -12,13 +12,14 @@ type nodeMessageProps = {
   text: string
   id: number
   key: string
+  type: string
```
Comment (jonas-jonas) [inline src/react-components/ory/helpers/error-messages.tsx:15]: With the new spec, we could get the types from the generated client.

Comment (Benehiko) [reply src/react-components/ory/helpers/error-messages.tsx:15]: I would need to bump the client version tho which could be a bit of a refactor. Let me do that in a separate branch

Comment (jonas-jonas) [reply src/react-components/ory/helpers/error-messages.tsx:15]: I actually already did both of those here: https://github.com/ory/elements/pull/56 and https://github.com/ory/elements/pull/57

But I needed to wait for the new client, which is released as of today. :)

Comment (jonas-jonas) [reply src/react-components/ory/helpers/error-messages.tsx:15]: You can either take them over, if you want to, or I can do it, once I have time^^ Not sure when that is tho :D

**Pull request merged**